### DISCUSSION TITLE
[BUG] Install script --upgrade default actions

### DISCUSSION THREAD

**Discussion by JReming85** [modoboa/modoboa#3079] [Installer]
# Impacted versions
Latest

Just ran the installer script to upgrade my Modoboa version and kept running into the same UI issue

First question
`It seems that your config file is outdated. Would you like to update it? (Y/n)`

Why have the Y capitalized when hitting enter defaults to N?  It should be reversed.

**Comment by Spitfireap** [comment]
Is it common to have the capitalized option to be default in a yes/no ? Here the idea was simply to covey the relative liberty regarding casing in the user input.

**Comment by JReming85** [comment]
Yes, I am surprised you even have to ask.  It has been the default to capitalize the default action ever since I first started using computers over 25-30 years ago.
